packages/procaptcha/src/modules/collector.ts

```diff
@@ -1,6 +1,6 @@
 import { ProsopoKeyboardEvent, ProsopoMouseEvent, ProsopoTouchEvent } from '@prosopo/types'
 
-const COLLECTOR_LIMIT = 1000
+const COLLECTOR_LIMIT = 10000
```

Comment:
we need to check what kind of db storage this takes. Is this the first 10k events which fire, then drop the rest?

Reply:
it's most recent 10k events - depends on polling rate of your mouse and how much you wiggle it

Reply:
okay and how big are those when fully generated? just wondering how much storage this will take really